surveys/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from . import views_api
+
+
+urlpatterns = [
+    url(r'^forms/(?P<pk>\d+)/$', views_api.SurveyView.as_view(), name='survey-form'),
```

Comment:
I think `angular` queries the urls without a `/` at the end and I think in general we make it optional with a `?`:
```python
    url(r'^forms/(?P<pk>\d+)/?$', views_api.SurveyView.as_view(), name='survey-form'),
```